std/http/server.ts

```diff
@@ -381,12 +381,25 @@ export class Server implements AsyncIterable<ServerRequest> {
     mux.add(this.acceptConnAndIterateHttpRequests(mux));
     return mux.iterate();
   }
+
+  // Returns the addres which this server is listening on.
+  get url(): string {
+    return `http://localhost:${this.listener.addr().address.split(':').pop()}/`
```

Comment:
Why is `localhost` hardcoded?

Reply:
I changed that in the latest commit

Reply:
because I only tested it with no hostname where the listener address defaults to `0.0.0.0` and `localhost` where the address becomes `[::1]`
so I thought it would be simpler to just use localhost.